### ISSUE TITLE
JPEG XL

### ISSUE THREAD

**Issue by tbs** [Lymphatus/libcaesium#13]
Any plan to support JPEG XL soon?

https://jpeg.org/jpegxl/
https://github.com/libjxl/libjxl
https://gitlab.com/wg1/jpeg-xl

**Comment by blopker** [comment]
Not sure if this is still on the radar, but there seems to be Rust bindings for libjxl: https://github.com/inflation/jpegxl-rs It has lossless and quality settings for encoding.

**Comment by blopker** [comment]
Yeah... That's frustrating. It seems like other wrapper crates work without issue. I wonder if this crate just isn't set up well, or there's something inherently complex about the underlying library.
